freqtrade/exchange/exchange.py

```diff
@@ -452,6 +452,17 @@ def price_to_precision(self, pair: str, price: float) -> float:
                 price = ceil(big_price) / pow(10, symbol_prec)
         return price
 
+    def price_get_one_pip(self, pair: str, price: float) -> float:
+        """
+        Get's the "1 pip" value for this pair.
+        Used in PriceFilter to calculate the 1pip movements.
+        """
+        precision = self.markets[pair]['precision']['price']
+        if self.precisionMode == TICK_SIZE:
+            return price % precision
+        else:
+            return 1 / pow(10, precision)
```

Comment:
it will not work for bitfinex/bitfinex2
(that exchange API uses the SIGNIFICANT_DIGITS as precisionMode property)

just a note for future devs

Reply:
I know, but the same "problem" exists for `price_to_precision()` - which is WAY more critical than this - and until we implement it there - i don't see a point in trying to support it here.